core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/event/project/CurrentProjectChangedEvent.java

```diff
@@ -26,16 +27,16 @@
     /** Type class used to register this event. */
     public static Type<CurrentProjectChangedHandler> TYPE = new Type<>();
 
-    private final ProjectConfigDto configDto;
+    private final Project project;
 
     /**
      * Creates an event to initiate changing of current project.
      *
-     * @param configDto
+     * @param project
      *         selected project
      */
-    public CurrentProjectChangedEvent(@NotNull ProjectConfigDto configDto) {
-        this.configDto = configDto;
+    public CurrentProjectChangedEvent(Project project) {
```

Comment:
if @Notnull is removed on constructor for Project, it should be removed from getter as well (or kept in constructor)